src/clover/scripts/hpc.py

```diff
@@ -91,6 +91,8 @@ def main(args: List[Any]) -> None:
     clover_arguments = [
         "--location",
         hpc_run.location,
+        "--output",
+        hpc_run.output,
```

Comment:
These two lines ensure that output file names are correctly translated from the `hpc_runs.yaml` input file to the CLOVER model